setup.py

```diff
@@ -46,6 +46,7 @@
         "fasttext==0.9.2",
         "pycld2==0.41",
         "justext==3.0.0",
+        "resiliparse",
```

Comment:
Do you have any concerns with not pinning the version?

Reply:
Not really. I can pin the most recent version that exists right now, but I don't anticipate any issues since we are only using a very basic Resiliparse function `extract_plain_text` one time. Let me know.

Reply:
Sounds good. I'm fine leaving it like this. It's probably worth adding a few small unit tests to ensure that parsing a basic HTML page works properly with our system though just so we can catch if a version change breaks things.